app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -1,197 +1,6 @@
 module EmsRefresh
   module Refreshers
     module EmsRefresherMixin
```

Comment:
Shouldn't this whole file just be deleted now?

Reply:
I wanted to leave it blank so we could remove all of the includes from providers then delete the file without leaving things in a broken state until all the provider PRs were merged.

Reply:
If we want to delete the file now I'll put PRs in to all the provider repos that include this mixin and can merge all at the same time.